client.go

```diff
@@ -26,11 +26,12 @@ func CallWithCodec(cc rpc.ClientCodec, method string, args interface{}, resp int
 	if err := cc.ReadResponseHeader(&response); err != nil {
 		return err
 	}
-	if err := cc.ReadResponseBody(resp); err != nil {
-		return err
-	}
 	if response.Error != "" {
+		cc.ReadResponseBody(nil)
```

Comment:
This ignores any errors produced by the body read to nowhere. Is there anything we could do if it fails? I guess we don't care, but it might mask weird things.